### ISSUE TITLE
Outdated snapshot dependencies for geotools?

### ISSUE THREAD

**Issue by ed-g** [pouyalireza/TNAST_MAVEN#1]
Output from `mvn compile` running in `TNAST_MAVEN/TNAtoolAPI-Webapp` directory:

> [ERROR] Failed to execute goal on project TNAtoolAPI-Webapp: Could not resolve dependencies for project TNAtoolAPI-Webapp:TNAtoolAPI-Webapp:war:0.0.1-SNAPSHOT: The following artifacts could not be resolved: org.geotools:gt-swing:jar:12-SNAPSHOT, org.geotools:gt-epsg-hsql:jar:12-SNAPSHOT, org.geotools:gt-shapefile:jar:12-SNAPSHOT, org.geotools:gt-referencing:jar:12-SNAPSHOT, org.geotools:gt-render:jar:12-SNAPSHOT, org.geotools:gt-opengis:jar:12-SNAPSHOT, org.geotools:gt-api:jar:12-SNAPSHOT, org.geotools:gt-main:jar:12-SNAPSHOT, org.geotools:gt-coverage:jar:12-SNAPSHOT, org.geotools:gt-data:jar:12-SNAPSHOT: Could not find artifact org.geotools:gt-swing:jar:12-SNAPSHOT in OSGEO GeoTools repo (http://download.osgeo.org/webdav/geotools) -> [Help 1]

Should we change the version of geotools to a newer version than 12-snapshot (from pom.xml):

```xml
  <properties>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
        <geotools.version>12-SNAPSHOT</geotools.version>
  </properties>
```

Or, was this version of geotools built locally? If so are there instructions to do this?

**Comment by ed-g** [comment]
I've upgraded to 12.5 in pom.xml and mvn compile now works. Does that seem reasonable to you?

```xml
  <properties>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
        <geotools.version>12.5</geotools.version>
  </properties>
```

**Comment by BenFields724** [comment]
The upgrade to 12.5 fixes the dependencies so this sounds reasonable. Are there additional issues other than this in getting your instance up and running?

**Comment by ed-g** [comment]
@BenFields724 great. Would you please update https://github.com/pouyalireza/TNAST_MAVEN/blob/master/TNAtoolAPI-Webapp/pom.xml and add a comment here with the commit number?

**Comment by pouyalireza** [comment]
This has been fixed commit number : 7e1015317249d12487784448fb8fa2fb7b3a5d1c

**Comment by ed-g** [comment]
Looks good, thanks!
